src/bundle/Resources/views/themes/admin/system_info/platform.html.twig

```diff
@@ -0,0 +1,67 @@
+{% trans_default_domain "systeminfo" %}
+
+{% set icons = asset('bundles/ezplatformadminui/img/ez-icons.svg') %}
```

Comment:
We are changing the way of adding icons to the twig, please see this: https://github.com/ezsystems/ezplatform-core/pull/21
I know this is `work in progress` but we will merge it as soon as possible so could you adapt to this and use the twig helper? `{{ ez_icon_path('checkmark') }}`